### PR TITLE
Create Make & Name 4 ships

### PR DESCRIPTION
**Pull request by joyminor**
It's not complete, but this is what I have so far. Please give me some feedback. Thanks.

### CODE REVIEW

File: Make & Name 4 ships [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+// Create a ship class with some attributes
+class Ship {
+  public:
+	string nameofShip;
+	char orientation;
+	int xCoor;
+	int yCoor;
+};
+
+int main() {
```
Comment (AlessandrielH) [inline Make & Name 4 ships:14]: int main is not used in class files.

File: Make & Name 4 ships [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  // Create a Ship 1
+  Ship shipPlacement1;
```
Comment (AlessandrielH) [inline Make & Name 4 ships:16]: shipPlacement is the name of the Ship Class

File: Make & Name 4 ships [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  shipPlacement1.nameofShip="Conquerer";
+  shipPlacement1.orientation;
```
Comment (AlessandrielH) [inline Make & Name 4 ships:18]: What did you intend to do on this line?  You need to create a function to set it

**Review by jgminor7 — APPROVED** [review]
:)

**Review by AlessandrielH — APPROVED** [review]

**Pull request merged**